### ISSUE TITLE
gmap.net winform show exception timeout

### ISSUE THREAD

**Issue by dicktwc** [judero01col/GMap.NET#191]
I use gmap.net component to show OSM map tile , but why my app show exception timeout string with blue color in PC, other PC is normal , I do'nt why

**Comment by petvetbr** [comment]
Same here, let me know if there is any progress on this.

**Comment by Triskh** [comment]
Yes, i also have the same problem.

**Comment by zgabi** [comment]
Probaby the same isse as in #188 #189 and #192

**Comment by Triskh** [comment]
Yes that could be the problem. I also found the url in the gmap.net.core but how can i edit the file?

**Comment by dicktwc** [comment]
I update win11 to the the latest , after some day, no exception in my application.
